ast_tools/passes/debug.py

```diff
@@ -5,17 +5,19 @@
 import astor
 
 from . import Pass
-from . import _PASS_ARGS_T
+from . import PASS_ARGS_T
 from ast_tools.stack import SymbolTable
 
+__ALL__ = ['debug']
+
 class debug(Pass):
     def __init__(self,
-            dump_ast: bool = True,
+            dump_ast: bool = False,
             dump_src: bool = False,
             dump_env: bool = False,
             file: tp.Optional[str] = None,
             append: tp.Optional[bool] = None,
-            ) -> _PASS_ARGS_T:
+            ) -> PASS_ARGS_T:
```

Comment:
A future improvement might be to have this configurable by the environment or config file, so by default it's quiet but for debugging we can easily set something to have a more useful log print out (e.g. passes can annotate places for a "log pass output" to easily see intermediate stages of the transformations.